README.md

```diff
@@ -137,40 +137,11 @@ To install the metamodels and run the tests (on unix-like system), you can do th
 	py.test 02_shared_grammar/tests/                   # build
```

Comment:
There is still a section `Note on the CLI for the three metamodels` in the README that should be reworked or removed as it reference old dsl compiler.

Reply:
Also search for `types_data_flow_dslc`. There are some refrerences in `.pu` images. Those need update too.

Reply:
fixed. I didn't think of them. Very good!